app/src/componentLibrary/systemMessage/systemMessage.scss

```diff
@@ -0,0 +1,58 @@
+@mixin font-config($weight, $size, $height, $color) {
+  font-weight: $weight;
+  font-size: $size;
+  line-height: $height;
+  color: $color;
+}
+
+@mixin message-headers($color) {
+  color: $color;
+  text-transform: capitalize;
+  margin-bottom: 8px;
+  font-weight: 500;
```

Comment:
font roboto with font-weight: 500 it is a $FONT-ROBOTO-MEDIUM
fw: 400 - $FONT-ROBOTO-REGULAR
fw: 500 - $FONT-ROBOTO-MEDIUM
fw: 700 - $FONT-ROBOTO-BOLD